tester/core/tests/mock_call.rs

```diff
@@ -946,5 +946,26 @@ async fn test_call_second_custom_enum_with_fields_arg_result() -> Result<(), Nov
 
     assert_eq!(result.result.unwrap(), expected);
 
+    Ok(())
+}
+
+
+#[tokio::test]
+async fn test_return_big_int_arg() -> Result<(), NovaXError> {
+    let executor = get_executor();
+
+    let result = TesterContract::new(
+        TESTER_CONTRACT_ADDRESS
+    )
+        .call(executor.clone(), 600000000)
+        .return_big_int_arg(&BigInt::from(42i8))
+        .await?
+        .result
+        .unwrap();
+
+    let expected = BigInt::from(42 as u8);
```

Comment:
Can you keep the same instantiation of BigInt you did elsewhere via i8? To keep consistency

Reply:
Sure, I missed that! All done